spring-cloud-netflix-core/pom.xml

```diff
@@ -6,7 +6,7 @@
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-netflix</artifactId>
 		<version>1.1.0.BUILD-SNAPSHOT</version>
-		<relativePath>..</relativePath>
+                <relativePath>..</relativePath> <!-- lookup parent from repository -->
```

Comment:
hurt me to use a tab.. suspecting now the ones with tabs are the wrong ones? :) Anyway I'll change to spaces with much delight!